docs/interfaces/pyapi/pyapi.rst

```diff
@@ -0,0 +1,425 @@
+.. highlight:: none
+.. _sec-interfaces-pyapi:
+
+#########################
+Python Interface - ctypes
+#########################
+
+ctypes is a foreign function library for Python. It provides C compatible data
+types, and allows calling functions in DLLs or shared libraries. It can be used
+to wrap these libraries in pure Python (cf. `ctypes documentation 
+<https://docs.python.org/3/library/ctypes.html>`_).
```

Comment:
Not sure if this is necessary or why the user should care about the `ctypes` internals.

Reply:
I think that to an acceptable degree one should always know what's going on under the hood, even if it's just technical details. But in reality that's probably not possible, so I'll just remove the short introduction :)

Reply:
I was following a similar strategy for my first `ctypes` API as well, from my experience, it was distracting / confusing most users.